### ISSUE TITLE
5/6 tests fail

### ISSUE THREAD

**Issue by MarkSwanson** [nlfiedler/magick-rust#2]
Everything compiles fine, but 5 out of 6 tests fail. Example:

---- test_read_from_blob stdout ----
	thread 'test_read_from_blob' panicked at 'assertion failed: wand.read_image_blob(data).is_ok()', tests/lib.rs:80
stack backtrace:
   1:     0x7f6dc27511e0 - sys::backtrace::tracing::imp::write::hdd9eea771561f811lbu
...

Note: test_new_drop() does succeed.

I tried with a Dec 21 nightly. Should I use a different Rust version?
I compiled/tested against ImageMagick libMagickCore-6.Q16.so.2.0.0

Thanks!

**Comment by nlfiedler** [comment]
@MarkSwanson Can you try checking out commit 34c4fec and seeing if that works? If so, then the problem lies with the recently added `build.rs` script, which apparently is generating a broken `bindings.rs` file for you. Otherwise, yes, I would recommend using the latest release of Rust. I know that both 1.5 and 1.6 will work fine.

**Comment by MarkSwanson** [comment]
Tried commit 34c4fec with rust 1.6 and it still fails.
Perhaps the latest v6 version of MagickCore is incompatible?
Which version of ImageMagic are you using? git sha?

**Comment by MarkSwanson** [comment]
Lol... I figured it out.
It's actually possible to build ImageMagick without jpeg or png support. In fact, a base Ubuntu 14 system will do just that.
Your library was working just fine all along (bindgen issues aside).

Steps to make things work on my system:
- sudo apt-get install libjpeg-dev
- sudo apt-get install libpng-dev
- then ./configure and build/install ImageMagick.

**Comment by nlfiedler** [comment]
@MarkSwanson Thanks for figuring this out. I tried it myself in a VM and found that {{sudo apt-get build-dep imagemagick}} gave me everything I needed to build ImageMagick with support for plenty of image formats. I found it very difficult to get PNG to be detected by the {{configure}} script by adding one PNG library or another; only the {{build-dep}} command worked for me. I'll add my steps to the README later today.
